assets/js/plugin-check-admin.js

```diff
@@ -25,17 +27,96 @@
 		);
 	} );
 
+	/**
+	 * Setup the runtime environment if needed.
+	 *
+	 * @since n.e.x.t
+	 */
+	function setupEnvironment( data ) {
```

Comment:
Nit-pick, but per my previous comment, this will also need to be changed accordingly:
```suggestion
	function setUpEnvironment( data ) {
```